guardian/manage-frontend
# Add annual membership new prices

## What does this change?

Add the post price rise Membership prices for annual memberships and removes the eligibility check from the save journey.



## How to test

Create an annual Membership - cancel, see the new journey (`/cancel/membership/landing`)



## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
